samples/bluetooth/central_uart/Kconfig

```diff
@@ -14,3 +14,11 @@ config ZMS
 
 config NVS
 	default y if !(SOC_FLASH_NRF_RRAM || SOC_FLASH_NRF_MRAM)
+
+config AUTO_CONNECT_BONDED_DEVICE
+	bool "Automatically connect to bonded device"
+	depends on SETTINGS && BT_MAX_CONN = 1
```

Comment:
Do we actually need the settings dependency here?

Reply:
out of date